Backend/TimeTracker.Api/Controllers/AuthController.cs

```diff
@@ -135,9 +136,23 @@ public async Task<GenericResponseDTO<int>> Register(UserDTO registerData)
                     CreatedTime = DateTime.UtcNow,
                     Email = registerData.Email,
                     Password = authHelper.GetPasswordHash(registerData.Password, configuration),
-                    Name = registerData.Name
+                    Name = registerData.Name,
+                    Projects = new List<Project>()
                 };
 
+                // check if the user registered with an invite code, if they did, add them to a project
+                if(registerData.InviteCode != null && !registerData.InviteCode.SequenceEqual("")) {
```

Comment:
There is a really nice C# function that can make this check much easier `String.IsNullOrWhiteSpace(registerData.InviteCode)`